content/en/tracing/connect_logs_and_traces/php.md

```diff
@@ -17,12 +17,12 @@ further_reading:
       text: 'Correlate request logs with traces automatically'
 ---
 
-## Automatic Trace ID injection
+## Automatic Injection into Logs
```

Comment:
Same as other languages, I would recommend to be more explicit in the title about what we inject or what we are doing at least